app/assets/javascripts/winners/charts.js

```diff
@@ -227,6 +227,16 @@
       }
     },
     create: {
+      // Creates a scatterplot with a connecting sparkline
```

Comment:
I agree that the names need some refactoring. It feels quite a bit outside the scope of this PR. Why don't I just remove the comment for now, and create a naming issue for later refactoring?